src/components/Tabs.js

```diff
@@ -47,19 +65,25 @@ class Tabs extends Component {
                 this.transformChildren(child.props && child.props.children, {
                     handleSelect,
                     selectedTab,
-                    firstLinkFound
+                    activeLinkStyle,
+                    name
                 })
             );
         });
     }
 
     render() {
         const handleSelect = this.props.handleSelect || this.handleSelect.bind(this);
-        const selectedTab = this.props.selectedTab || this.state.selectedTab;
+        const selectedTab = this.props.selectedTab ||
+            this.state.selectedTab ||
+            this.state.defaultTab ||
```

Comment:
I don't think `defaultTab` in state is necessary. If `this.props.selectedTab` and `this.state.selectedTab` are not defined/null, we can set state of `selectedTab` to the value returned by `this.findDefault(this.props.children)`